cmd/main.go

```diff
@@ -107,12 +108,18 @@ func main() {
 		Scheme: mgr.GetScheme(),
 	}
 
+	loggingSecrets := loggingcredentials.Reconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
```

Comment:
I think you are not using the `Scheme` field, so it could be removed from the reconciler.

Reply:
Done.